src/modules/Root/Root.jsx

```diff
@@ -10,27 +10,36 @@ import Header from '../Layout/Header/Header';
 import Footer from '../Layout/Footer/Footer';
 import APIRespondModalContainer from '../modals/APIRespondModal/apiRespondModalContainer';
 
+import LoginPage from '../pages/LoginPage/LoginPage';
 import ApiListPage from '../pages/ApiListPage/ApiListPage';
 import NewApiPage from '../pages/NewApiPage/NewApiPage';
 import EditApiPage from '../pages/EditPage/EditApiPage';
 
+import block from '../../helpers/bem-cn';
+
+import { navClassName } from '../Layout/Nav/Nav';
```

Comment:
From one side it would be more logical to make all NavLinks within Nav component. But in that case it would be a bit harder to track the routes paths, since one part (`NavLink` with e.g. `to="/new"`) would be living inside `Nav` and `<Route path="/new"` - in `Root` component. Thant's why I've decided to place everything (`NavLink`s and `Route`s) in one place.
And because of that decision, I need to export navigation class from `Nav` to be able to set proper className to `NavLink` in `Root.jsx`